### DISCUSSION TITLE
Emoji Event Trigger for Github Actions

### DISCUSSION THREAD

**Discussion by aguckenber-chwy** [community/community#37901] [Actions]
Would it be possible to add a trigger which fires a Github Action when an Emoji is made? This would make it so actions can be programmed to add comments to a PR and ask for permission to continue while using a ✅  or a ❌  to ask developers an option (the action can have code to make sure the users who click the emoji are authorized). This would be super handy! Maybe this is already supported and I am just missing it in the documentation.

**Comment by ossanna16** [comment]
Hi there @aguckenber-chwy and welcome to our community! Thank you for the feature request!

**Comment by bernatfp** [comment]
YES! This would make the experience of approving certain steps of a workflow in a PR much easier. Requiring a comment to be added in a certain way is a bit tedious and error prone.

**Reply by rdhar** [reply]
Exactly the same use-case here.

To give an example, it'd be ideal to to add a 🚀 emoji to Terraforms plan output comment and have that event trigger a workflow to Terraform apply.

**Comment by ChrisWiles** [comment]
I am trying to implement a GitHub Action workflow that ensures the author of a pull request acknowledges suggestions made by a PR bot by adding an emoji reaction to the bot's comment. I need a Emoji event trigger
